docs/tutorials/custom-cluster-selector.md

```diff
@@ -0,0 +1,637 @@
+---
+layout: default
+title: "Customizing Kubernetes YAMLs to target specific cluster"
+permalink: /tutorials/customize-cluster-selector
+parent: Tutorials
+nav_order: 9
+---
+
+# Customizing Kubernetes YAMLs to target specific cluster
+
+## Description
+
+Move2Kube already supports targeting amongst multiple Kubernetes cluster, like- Kubernetes, Openshift, IBM-IKS, IBM-Openshift, Azure-EKS, Azure-AKS and GCP-GKS, but there might be situations where you might require generating Kubernetes YAMLs to target particular cluster. In this tutorial we will see how we can use Konveyor Move2Kube to change the versions of existing Kubernetes resources to target a particular cluster. Move2Kube can also be customized to generate Kubernetes YAMLS deployable on a particular cluster.
+
+## Prerequisites
+
+1. Install the [Move2Kube CLI tool](https://move2kube.konveyor.io/installation/cli)
+
+1. Install [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl)
+
+1. We will use [kubernetes-to-kubernetes](https://github.com/konveyor/move2kube-demos/tree/main/samples/kubernetes-to-kubernetes) sample. The `kubernetes-to-kubernetes` directory has some Kubernetes YAMLs that deploy a web app with multiple services. There are 3 services: a frontend website in PHP, a backend API in NodeJS and a cache service using Redis.
+
+## Steps
+
+1. Let's first collect the data about your Kubernetes cluster using `move2kube collect`. Here we will limit the collect to only cluster information using the `-a k8s` annotation flag. Please note: Before running the below command, log in to your target cluster.
```

Comment:
We need to mention about the need to have the cluster in context for kubectl

Reply:
Done.